CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Commit hash to version output.
```

Comment:
Now that we show `tag-ahead-ghash` in the version output, this doesn't seem as accurate anymore. What do you think about something like:

```suggestion
- Version output now displays `git` revision information.
```

Reply:
Sounds good to me!

Reply:
(I dropped "now" because it's under the `Added` header -- it's a given this didn't happen before.)